test/extended/util/prometheus/helpers.go

```diff
@@ -444,3 +498,9 @@ func ForEachAlertingRule(rules map[string][]promv1.AlertingRule, f func(a promv1
 
 	return fmt.Errorf("Incompliant rules detected:\n\n%s", strings.Join(allViolations.List(), "\n"))
 }
+
+// url.JoinPath but with no error checks
+func JoinUrl(base string, paths ...string) string {
+	path, _ := url.JoinPath(base, paths...)
```

Comment:
For safety, it would better to panic if an error is returned (which should never happen)
```suggestion
	path, err := url.JoinPath(base, paths...)
	if err != nil {
		panic(err)
	}
```

Reply:
Right, like a `throw()` :D